resources.html

```diff
@@ -90,6 +90,14 @@ <h5>At a glance</h5>
             </ul>
         </div>
 
+        <h5>Visualisation of kkrunchy report</h5>
+
+        <p>
+            A typical difficulty when working with a compression tool, is understanding how code and data impacts the final binary size.<br/>
+            <a href="https://github.com/ConspiracyHu/kkpView-public">KKP Analyzer</a> is a tool to analyse and visualise the compression information provided by kkrunchy. It is written by BoyC of Conspiracy.
```

Comment:
I see several reasons to keep them: it can draw attention when a same author appears several times, it helps find information ("didn't xyz make a thing for texture"), and it's a courtesy as well.